assets/css/style.scss

```diff
@@ -223,7 +223,7 @@ div.breadcrumbs.bootstrap .breadcrumb-item + .breadcrumb-item:hover::before {
 
 h2 a,
 .api code {
-      white-space: nowrap;
+      white-space: pre;
```

Comment:
I had always assumed that was there for a reason. @kent-mcleod, @wom-bat do you remember anything on that?

Reply:
Looks like it was added ~4 years ago in 09707fd420b1996b96ebb722a58afba7c8c7976d by @WellMcGarnicle (though I certainly doubt I would remember any particulars)

Reply:
This style is for API docs to stop code sections from wrapping. The style was guarded by an api class inherited by using the API layout.  At some point docs like GettingStarted started also using the API layout in order to get a table-of-contents on the side.  It seems like `pre` is an improvement though.

Reply:
(The original issue was that the invocation names on https://docs.sel4.systems/projects/sel4/api-doc.html would get broken up when the text wrapped on mobile screens I think).

Reply:
Ok, then I think `pre` is indeed an improvement.